code/model/modelcollide.cpp

```diff
@@ -558,6 +615,7 @@ int model_collide_sub(void *model_ptr )
 				return 1;
 			}
```

Comment:
Looks like the tabs got knocked out of alignment here at some point.

Reply:
Which tabs?

Reply:
Ok, I checked out the branch locally and looked at it again with the local editor.  Seems the first time I read the diff on GitHub I lost track of what brace went where.